sgkit_plink/tests/test_pysnptools.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+from sgkit_plink import pysnptools
```

Comment:
I don't think users should have to know about the existence of `pysnptools` to use the library. Can we just expose the `read_plink` API so you can do:

    from sgkit_plink import read_plink

Reply:
Makes sense.  I'm assuming you're thinking that if we had two plink readers then we have an `engine` argument or something of the like?

Reply:
Changed in https://github.com/pystatgen/sgkit-plink/pull/1/commits/717d307dab6f4bd35088cb3896ca84d65c312256